cni/pkg/iptables/testdata/default.golden

```diff
@@ -0,0 +1,16 @@
+iptables -t mangle -N ISTIO_PRERT
+iptables -t nat -N ISTIO_OUTPUT
+iptables -t mangle -N ISTIO_OUTPUT
+iptables -t mangle -A PREROUTING -j ISTIO_PRERT
+iptables -t mangle -A OUTPUT -j ISTIO_OUTPUT
+iptables -t nat -A OUTPUT -j ISTIO_OUTPUT
+iptables -t mangle -A ISTIO_PRERT -m mark --mark 0x3/0xfff -j CONNMARK --set-xmark 0x111/0xfff
+iptables -t mangle -A ISTIO_PRERT -s 169.254.7.127 -p tcp -m tcp -j ACCEPT
+iptables -t nat -A ISTIO_OUTPUT -d 169.254.7.127 -p tcp -m tcp -j ACCEPT
```

Comment:
what is this for

Reply:
See: https://github.com/istio/istio/pull/48253/files#diff-e979dbe9201b8c8f280cbe55ba0bb7ebf7c5aaf1e63134ecb697e794c9c538d1R190

And also related discuss about it here: https://github.com/istio/istio/pull/48253#discussion_r1423068271